src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -368,6 +375,76 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 5,
                 "Current action does not explicitly specify any content types for the response.");
+
+            _selectingOutputFormatterUsingAcceptHeader = LoggerMessage.Define(
+                LogLevel.Debug,
+                6,
+                "Attempting to select an output formatter based on Accept header.");
```

Comment:
Should we log the accept header value here?

Reply:
Ah yes, we should. I didn't realize hosting does not log the accept header as Info where as we do for content-type.